__tests__/db.test.js

```diff
@@ -174,4 +174,73 @@ describe("/api/reviews/review:id/comments", () => {
         expect(result.body.msg).toBe("Invalid endpoint input");
       });
   });
+
+  test("POST - status: 201 - should add comment and return the new comment object", () => {
```

Comment:
We need another test for what happens when there are unnecessary extra properties on the comment - they should be ignored and the comment posted, as long as the rest of the comment is fine